iDiff/cmd/iDiff.go

```diff
@@ -98,4 +99,5 @@ func checkArgType(args []string) (bool, error) {
 func init() {
 	RootCmd.AddCommand(iDiffCmd)
 	iDiffCmd.Flags().BoolVarP(&json, "json", "j", false, "JSON Output defines if the diff should be returned in a human readable format (false) or a JSON (true).")
+	iDiffCmd.Flags().BoolVarP(&eng, "eng", "e", false, "By default the docker calls are shelled out locally, set this flag to use the Docker Engine Client (version compatibility required).")
```

Comment:
If you set the client API version properly, you shouldn't need to shell out.  Then, you can use the client library everywhere.